### ISSUE TITLE
Crashes when using ls if entry extension returns None value in rec_list

### ISSUE THREAD

**Issue by Zrocket** [shashwatah/jot#12]
When playing around with jot using my existing Obsidian vault this error would occour when trying to list the directory:

```
Notes
├── Notes
│   ├── Bins
│   │   └── Images
│   │       └── LYT Kit
│   ├── .obsidian
thread 'main' panicked at /home/tamer/.cargo/registry/src/index.crates.io-6f17d22bba15001f/jt-0.1.2/src/utils.rs:188:49:
called `Option::unwrap()` on a `None` value
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

Poking around in the source, I found that this is a result of line 199 in utils.rs:

`        if entry.is_file() && entry.extension().unwrap() != "md" {
`

Specifically, this is a result of trying to unwrap a None value if an extension is not found.

If it pleases you, I've created a simple fix that resolves the issue.

**Comment by Zrocket** [comment]
I created a pull request.

**Comment by shashwatah** [comment]
I've merged your pull request (#13). 
Thanks for the contribution!
